README.md

```diff
@@ -1,2 +1,30 @@
-# scraped_page
+# ScrapedPage
+
 Write declarative scrapers in Ruby
+
+## Usage
+
+Create a subclass of `ScrapedPage` for each page you wish to scrape.
+
+```ruby
+require 'scraped_page'
+
+class ExamplePage < ScrapedPage
+  field :title do
+    noko.at_css('h1').text
+  end
+  
+  field :more_information do
+    noko.at_css('a')[:href]
+  end
+end
+
+puts ExamplePage.new('http://example.com').to_h
```

Comment:
I guess one key decision we need to make early is whether the URL is a bare parameter like this, or whether we go for a named parameter for it.